src/react-components/media-browser.js

```diff
@@ -282,6 +282,13 @@ class MediaBrowser extends Component {
     const activeFilter =
       searchParams.get("filter") || (searchParams.get("similar_to") && "similar") || (!searchParams.get("q") && "");
 
+    const meta = this.state.result && this.state.result.meta;
+    const hasNext = !!(meta && meta.next_cursor);
+    const hasPrevious = searchParams.get("cursor");
+    const page = (meta && meta.page) || 0;
+    const apiSource = (meta && meta.source) || null;
+    const isVariableWidth = ["bing_images", "tenor"].includes(apiSource);
```

Comment:
i'd suggest pushing api source as a prop and moving the variable width concern back into the media tiles component, since it's a rendering concern